docs/zkapps/tutorials/01-hello-world.mdx

```diff
@@ -97,87 +100,103 @@ The zkApp CLI will ask if you would like to create an acompnanying ui project. S
+
 ### Building and running
 
 :::info
 
-The following commands are included for reference, and will fail if you run them at this point. This is because we haven't written the `Square` smart contract yet.
+The following commands are included for reference and will fail if you run them now. This is expected behavior because you haven't written the `Square` smart contract yet.
 
 :::
 
-To compile the TypeScript code into JavaScript, and run the JavaScript code, you'll type:
+To compile the TypeScript code into JavaScript and run the JavaScript code:
 
 ```
 $ npm run build
 $ node build/src/main.js
 ```
 
-The first line creates JavaScript code in the build directory, while the second line runs the code in `src/main.ts`.
+The first command creates JavaScript code in the build directory. The second command runs the code in `src/main.ts`.
 
 You can also combine these together into one line, such as:
 
 ```
-$ npm run build && node build/src/main.js
+npm run build && node build/src/main.js
 ```
 
-This will run `main` if the build is successful.
+This command runs `main` if the build is successful.
 
 ## Write the zkApp Smart Contract
 
-Now, the fun part! Let's write our smart contract: `src/Square.ts`. Line numbers are provided for convenience. A final version of what we're writing can be found [here](https://github.com/o1-labs/docs2/blob/main/examples/zkapps/01-hello-world/src/Square.ts).
+Now, the fun part! Write your smart contract, the `src/Square.ts` file. 
 
-:::tip
+Line numbers are provided for convenience. A final version of what you're writing is provided in the [Square.tx](https://github.com/o1-labs/docs2/blob/main/examples/zkapps/01-hello-world/src/Square.ts) example file.
 
-This tutorial will walk through the `Square` smart contract linked above. We recommend that you copy and paste the entire file from that link, and then follow the tutorial with the finished code in place.
+:::tip
 
-You can also copy and paste the code snippets into your project as you go - to avoid inserting the line numbers into your smart contract, copy these code snippets using the button provided. It will appear at the top right of the snippet box when you mouseover it.
+This tutorial walks through the `Square` smart contract code in the `src/Square.ts` as viewed in a text editor. If this is your first project, use the provided example file for the best user experience.  
 
 :::
 
+## Copy the example 
+
+The easiest way to walk through the example is to use the entire contents in the provided [Square.tx](https://github.com/o1-labs/docs2/blob/main/examples/zkapps/01-hello-world/src/Square.ts) example file. You can then follow the tutorial with the finished code in place.
+
+Or you can follow the steps to copy and paste the code snippets into your project as you go. To avoid inserting the line numbers into your smart contract, use the copy code to clipboard button that appears at the top right of the snippet box when you hover over it.
```

Comment:
hmmmm 
I am not clear on the value of giving two options to complete our first tuto...

how many users want to copy code in bits? or should we instead provide only one option to use the provided example contents?